plugins/inventory/proxmox.py

```diff
@@ -0,0 +1,350 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016 Guido Günther <agx@sigxcpu.org>, Daniel Lobato Garcia <dlobatog@redhat.com>
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: proxmox
+    plugin_type: inventory
+    short_description: proxmox inventory source
+    version_added: "1.1.0"
+    author:
+        - Jeffrey van Pelt (@Thulium-Drake) <jeff@vanpelt.one>
+    requirements:
+        - requests >= 1.1
+    description:
+        - Get inventory hosts from the proxmox service.
+        - "Uses a configuration file as an inventory source, it must end in ``.proxmox.yml`` or ``.proxmox.yaml`` and has a ``plugin: proxmox`` entry."
+        - Will retrieve the first network interface with an IP for Proxmox nodes
+        - Can retrieve LXC/QEMU configuration as facts
+    extends_documentation_fragment:
+        - inventory_cache
+    options:
+      plugin:
+        description: The name of this plugin, it should always be set to 'proxmox' for this plugin to recognize it as it's own.
+        required: True
+        choices: ['proxmox']
+        type: str
+      url:
+        description: url to proxmox
+        default: 'http://localhost:8006'
+        type: str
+      user:
+        description: proxmox authentication user
```

Comment:
```suggestion
        description: proxmox authentication user.
```
(Assuming proxymox has to be written lowercase.) Same for the next ones.

Reply:
I think it should start from a capital latter everywhere. https://www.proxmox.com/en/proxmox-ve they use "Proxmox", so it would be ok